packages/@aws-cdk/aws-eks-v2-alpha/lib/cluster.ts

```diff
@@ -1109,6 +1210,25 @@ export class Cluster extends ClusterBase {
         authenticationMode: 'API',
         bootstrapClusterCreatorAdminPermissions: props.bootstrapClusterCreatorAdminPermissions,
       },
+      computeConfig: {
+        enabled: !autoModeEnabled ? false : true,
+        // If the computeConfig enabled flag is set to false when creating a cluster with Auto Mode,
+        // the request must not include values for the nodeRoleArn or nodePools fields.
+        nodePools: !autoModeEnabled ? undefined : props.compute?.nodePools ?? ['system', 'general-purpose'],
+        nodeRoleArn: !autoModeEnabled ? undefined : props.compute?.nodeRole?.roleArn ?? this.addNodePoolRole(`${id}nodePoolRole`).roleArn,
```

Comment:
If `computeConfig` is specified while `autoMode` is disabled, does it give an error, a warning or just ignoring the property?

Reply:
I'll add a test case for that but technically when `computeConfig.enabled: false` you are still allowed to specify `nodePools` and `nodeRoleArn`. They are just being ignored.

Reply:
Well, I think it's weird to have

`defaultCapacityType.NODEGROUP`
with `computeConfig.nodePools` or `nodeRole` defined 
Let's throw and block them.

Reply:
DONE with more tests added.